src/jvm/main/org/jetbrains/kotlinx/lincheck/LinChecker.kt

```diff
@@ -130,6 +132,12 @@ class LinChecker (private val testClass: Class<*>, options: Options<*, *>?) {
             RandomProvider::class.java
         ).newInstance(this, testStructure, randomProvider)
 
+    private fun checkAtLeastOneMethodIsMarkedAsOperation(testClass: Class<*>) {
+        require (testClass.methods.any { it.isAnnotationPresent(Operation::class.java) }) {
+            "At least one method in a tested class should be marked as @Operation to make random scenarios generation possible. Please see official guide for more details."
```

Comment:
Soon, we'll provide an API that will not use annotations, and functions will be passed by reference. Please generalize the message correspondingly.